raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -763,6 +755,11 @@ contract TokenNetwork is Utils {
             returned_tokens
         );
 
+        // After the channel is settled the storage is cleared, therefor the
+        // value will be Gone and not Settled. The value Settled is used for
+        // the external APIs
+        require(channels[channel_identifier].state == ChannelState.NonExistent);
```

Comment:
@loredanacirstea This was removed by another PR [ https://github.com/raiden-network/raiden-contracts/commit/e8998ca1e66423b625755ac86e200acfebf2daaf#diff-2f6fb78ac7aaf29aaec1ed8ab59468f8L506 ], should I remove it from here too?

Reply:
You can leave it.